tests/evaluation-load-test/evaluation-load-test.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net8.0;net7.0</TargetFrameworks>
+    <TargetFrameworks>net7.0;net8.0</TargetFrameworks>

```

Comment:
.NET 7.0 is out of support so no need to test for releases that have yet to happen. Perhaps a separate PR should make this be net8.0 and net9.0 (I dont think net9.0 is necessarily required, but it would be more useful/relavant than net7) ?

Reply:
Agreed, I have removed net7.0 from the targetframeworks.